### PR TITLE
chore: standardize naming, move files to locale dir

### PR DESCRIPTION
**Pull request by mikejgray**

### CODE REVIEW

**Comment by mikejgray** [comment]
```
FAILED action/github/test/test_skill_resources.py::TestSkillResources::test_skill_setup - AttributeError: 'EasterEggsSkill' object has no attribute '_core_lang'. Did you mean: 'core_lang'?
```

This looks like a bug in the test, not so much the skill itself... `grep -R '_core_lang' .` returns nothing.

File: __init__.py [hunk 1]
```diff
@@ -16,17 +16,17 @@ class EasterEggsSkill(OVOSSkill):
     def grandma_mode(self):
         return self.settings.get("grandma_mode_enabled", True)
 
-    @intent_handler(IntentBuilder("GrandmaModeIntent").require("GrandmaMode").build())
+    @intent_handler(IntentBuilder("grandma_mode_intent").require("grandma_mode_keyword").build())
```
Comment (JarbasAl) [inline __init__.py:19]: not an issue for me, but historically we have used CamelCase in adapt, that convention you are migrating towards has been mainly for intent files

either is valid, we kinda just went with was was common in mycroft and never gave it much thought

Comment (mikejgray) [reply __init__.py:19]: I don't have a horse in the race so I'm happy to standardize on whatever makes sense. There's not really a set standard for the other either, but I think generally people like either dotted notation or snake_case.

What do you think? Is it worth separating Adapt out as its own naming convention? To me, an intent is an intent

Comment (JarbasAl) [reply __init__.py:19]: no strong opinion, and code wise its all valid, so its up to you! if we ever standardize this its one more thing we need to refactor around all repos and we can revisit then

Comment (mikejgray) [reply __init__.py:19]: In that case, laziness wins - I'll leave it as-is 😉

**Review by emphasize — APPROVED** [review]

**Pull request merged**